cypresse2e/cypress/integration/e2eTests/helpBoard/helpBoard.spec.js

```diff
@@ -0,0 +1,47 @@
+import HelpBoard from '../../../elements/pages/helpBoard'
+
+describe('FightPandemics Help Board Page for unauthorized user', () => {
+
+    const helpBoard = new HelpBoard();
+    var h1Heading = "Help Board";
+    var noPostsAvailableText = "Sorry, there are currently no relevant posts available. Please try using a different filter search or";
+
+    context('User opens Help Board', () => {
+        beforeEach(() => {
+            helpBoard.visit();
+        });
+
+
+        it('FP logo is visible and clickable', () => {
+            cy.checkFpLogoIsVisibleAndClickable(helpBoard.getFpLogoLocator());
+
+        });
+        
+        it('Help Board page contains heading', () => {          
+            cy.pageContainsHeading(helpBoard.getHelpBoardPageTitleLocator(),h1Heading);
+        });
+
+        it('Unauthorized user is redirected to SignIn page when clicking Add Post Button', () => {          
+            var addPostButton = helpBoard.getAddPostButton();
+            addPostButton.click({ force: true });
+            cy.validateCorrectScreenIsOpen("auth/login");
+        });
+
+        it('Unauthorized user sees a message that posts are not available - Help Board is empty', () => {     
```

Comment:
Maybe I am wrong but as I understand this test. If we want to test that unauthorized user sees a message that posts are not available then I think no need of these lines in the code :  

```
var addPostLink = helpBoard.getAddPostLinkElement();
                    addPostLink.click({ force: true });
                    cy.validateCorrectScreenIsOpen("auth/login"); 
```

and in the part which defines the length of the posts should be length = 0, otherwise this test is going to be always passed. The else part is unnecessary as we would like to test only the situation when there is no post available. Do I understand it right?

Reply:
I redesigned this test and marked it as skip.
This situation can happen using localhost.
![Screen Shot 2020-10-16 at 14 50 24](https://user-images.githubusercontent.com/21987083/96260472-59852400-0fbf-11eb-851b-9722c40e801c.png)

Reply:
Ok, I can see this... alright.